### PR TITLE
Elastic indices 4

### PR DESCRIPTION
**Pull request by miiu96**
#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [x] document new feature
- [ ] update documentation that is not relevant anymore
- [ ] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/sdk-and-tools/indices/delegators.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+
+The `_id` field of this index is composed in this way: `blake2bHash(delegatorAddress+stakingProviderAddress)`, in a base64 encoding (example: `YZNG+r3ZwFtOj0c057MnpVnXAfmSqLai15lusLWg+KM=`).
+
+## Fields
+
+
+| Field          | Description                                                                                                                          |
+|----------------|--------------------------------------------------------------------------------------------------------------------------------------|
+| address        | The address field holds the address in a bech32 encoding of the delegator.                                                           |
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/indices/delegators.md:17]: The address fields holds the bech32 encoded address of the delegator

File: docs/sdk-and-tools/indices/delegators.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+| contract       | The contract field holds the address in a bech32 encoding of the staking provider contract.                                          |
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/indices/delegators.md:18]: The contract field holds the bech32 encoded address of the staking provider contract the address delegated to.

File: docs/sdk-and-tools/indices/delegators.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+| activeStake    | The activeStake field holds the EGLD amount of the active stake (not undelegated, unbondable).                                       |
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/indices/delegators.md:19]: `not undelegated or unbondable` - sounds better. also for the next line

File: docs/sdk-and-tools/indices/logs.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+|-------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| identifier  | The identifier field represents the identifier of the event.                                                                                                                              |
+| address     | The address field holds the address in a bech32 encoding. It can be the address of the smart contract that generated the event or the address of the receiver address of the transaction. |
+| topics      | The topics field holds a list with extra information. They don't have a specific order because the smart contract is free to log anything that could be helpful.                          |
+| data        | The data field can contain information added by the smart contract that generates the event.                                                                                              |
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/indices/logs.md:28]: generates -> generated

File: docs/sdk-and-tools/indices/tags.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+## Fields
+
+| Field | Description                                                                          |
+|-------|--------------------------------------------------------------------------------------|
+| count | The count field represents the number of NFTs that have the tag from the tag field.  |
+| tag   | The tag field contains the tag in alphanumeric format.                               |
+
+## Query examples
+
+### Fetch the number of NFTs with the given tag.
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/indices/tags.md:20]: Fetch NFTs count with a given tag

File: docs/sdk-and-tools/indices/delegators.md (hunk 2 above)
Comment (miiu96) [reply docs/sdk-and-tools/indices/delegators.md:18]: done

File: docs/sdk-and-tools/indices/delegators.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+| activeStakeNum | The activeStake field holds the EGLD amount of the active stake (not undelegated, unbondable), in a numeric format. . Example: 1.5.  |
+
+## Query examples
+
+### Fetch all records for an address
```
[the remark this answers is not in the document]
Comment (miiu96) [reply docs/sdk-and-tools/indices/delegators.md:24]: changed

File: docs/sdk-and-tools/indices/tags.md (hunk 5 above)
Comment (miiu96) [reply docs/sdk-and-tools/indices/tags.md:20]: done

File: docs/sdk-and-tools/indices/tags.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+## _id
+
+The `_id` field of this index is represented by the tag name in a base64 encoding.
+
+## Fields
+
+| Field | Description                                                                          |
+|-------|--------------------------------------------------------------------------------------|
+| count | The count field represents the number of NFTs that have the tag from the tag field.  |
```
[the remark this answers is not in the document]
Comment (miiu96) [reply docs/sdk-and-tools/indices/tags.md]: done

File: docs/sdk-and-tools/indices/delegators.md [hunk 8]
```diff
@@ -12,16 +12,16 @@ The `_id` field of this index is composed in this way: `blake2bHash(delegatorAdd
...
-| address        | The address field holds the address in a bech32 encoding of the delegator.                                                           |
-| contract       | The contract field holds the address in a bech32 encoding of the staking provider contract.                                          |
-| activeStake    | The activeStake field holds the EGLD amount of the active stake (not undelegated, unbondable).                                       |
-| activeStakeNum | The activeStake field holds the EGLD amount of the active stake (not undelegated, unbondable), in a numeric format. . Example: 1.5.  |
+| Field          | Description                                                                                                                           |
+|----------------|---------------------------------------------------------------------------------------------------------------------------------------|
+| address        | The contract field holds the bech32 encoded address of the staking provider contract the address delegated to.                        |
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/indices/delegators.md:17]: revert

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]

File: docs/sdk-and-tools/indices/delegators.md (hunk 8 above)
Comment (miiu96) [reply docs/sdk-and-tools/indices/delegators.md]: done

File: docs/sdk-and-tools/indices/delegators.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+
+
+The `_id` field of this index is composed in this way: `blake2bHash(delegatorAddress+stakingProviderAddress)`, in a base64 encoding (example: `YZNG+r3ZwFtOj0c057MnpVnXAfmSqLai15lusLWg+KM=`).
+
+## Fields
+
+
+| Field          | Description                                                                                                                           |
+|----------------|---------------------------------------------------------------------------------------------------------------------------------------|
+| address        | The contract field holds the bech32 encoded address of the staking provider contract the address delegated to.                        |
```
[the remark this answers is not in the document]
Comment (miiu96) [reply docs/sdk-and-tools/indices/delegators.md]: is correct, changed

File: docs/sdk-and-tools/indices/delegators.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+| contract       | The contract field holds the address in a bech32 encoding of the staking provider contract.                                           |
+| activeStake    | The activeStake field holds the EGLD amount of the active stake (not undelegated or unbondable).                                      |
```
[the remark this answers is not in the document]
Comment (miiu96) [reply docs/sdk-and-tools/indices/delegators.md:19]: changed

File: docs/sdk-and-tools/indices/delegators.md [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+| activeStakeNum | The activeStake field holds the EGLD amount of the active stake (not undelegated or unbondable), in a numeric format. . Example: 1.5. |
```
[the remark this answers is not in the document]
Comment (miiu96) [reply docs/sdk-and-tools/indices/delegators.md:20]: changed

File: docs/sdk-and-tools/indices/delegators.md [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+  --data '{
+	"query": {
+		"match": {
+			"address":"erd.."
+		}
+	}
+}'
+```
+
+### Fetch all delegators from a staking provider
```
[the remark this answers is not in the document]
Comment (miiu96) [reply docs/sdk-and-tools/indices/delegators.md:39]: change

File: docs/sdk-and-tools/indices/logs.md [hunk 13]
```diff
@@ -0,0 +1,74 @@
...
+| originalTxHash | The originalTxHash field holds the hex-encoded hash of the initial transaction. When this field is not empty the log is generated by a smart contract result and this field represents the hash of the initial transaction. |
+| timestamp      | The timestamp field represents the timestamp of the block in which the log was generated.                                                                                                                                   |
+
+
+Event structure 
+
+| Field       | Description                                                                                                                                                                               |
+|-------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| identifier  | The identifier field represents the identifier of the event.                                                                                                                              |
```
[the remark this answers is not in the document]
Comment (miiu96) [reply docs/sdk-and-tools/indices/logs.md:25]: changed

File: docs/sdk-and-tools/indices/tags.md [hunk 14]
```diff
@@ -0,0 +1,33 @@
...
+## _id
+
+The `_id` field of this index is represented by the tag name in a base64 encoding.
+
+## Fields
+
+| Field | Description                                                          |
+|-------|----------------------------------------------------------------------|
+| count | The count field represents the number of NFTs with the current tag.  |
+| tag   | The tag field contains the tag in alphanumeric format.               |
```
[the remark this answers is not in the document]
Comment (miiu96) [reply docs/sdk-and-tools/indices/tags.md:16]: changed.

File: docs/sdk-and-tools/indices/operations.md [hunk 15]
```diff
@@ -0,0 +1,60 @@
...
+The unified structure will contain an extra field in order to be able to differentiate between them.
+
+| Field | Description                                                                                    |
+|-------|------------------------------------------------------------------------------------------------|
+| type  | It can be `normal` in case of a transaction and `unsigned` in case of a smart contract result. |
+
+
+## Query examples
+
+### Fetch the latest operations of an address
```
[the remark this answers is not in the document]
Comment (miiu96) [reply docs/sdk-and-tools/indices/operations.md:24]: In this example there you have to put the same address in order to fetch all the operations

File: docs/sdk-and-tools/indices/delegators.md [hunk 16]
```diff
@@ -12,12 +12,12 @@ The `_id` field of this index is composed in this way: `blake2bHash(delegatorAdd
...
-| contract       | The contract field holds the address in a bech32 encoding of the staking provider contract.                                           |
-| activeStake    | The activeStake field holds the EGLD amount of the active stake (not undelegated or unbondable).                                      |
-| activeStakeNum | The activeStake field holds the EGLD amount of the active stake (not undelegated or unbondable), in a numeric format. . Example: 1.5. |
+| Field          | Description                                                                                                                          |
+|----------------|--------------------------------------------------------------------------------------------------------------------------------------|
+| address        | The address field holds the address in a bech32 encoding of the delegator.                                                           |
+| contract       | This field holds the bech32 encoded address of the staking provider contract the address to whom it was delegated to.                |
```
Comment (mariusmihaic) [inline docs/sdk-and-tools/indices/delegators.md:18]:
```suggestion
| contract       | This field holds the bech32 encoded address of the staking provider contract to whom it was delegated to.                |
```

File: docs/sdk-and-tools/indices/tags.md [hunk 17]
```diff
@@ -10,10 +10,10 @@ The `_id` field of this index is represented by the tag name in a base64 encodin
...
 ## Fields
 
-| Field | Description                                                          |
-|-------|----------------------------------------------------------------------|
-| count | The count field represents the number of NFTs with the current tag.  |
-| tag   | The tag field contains the tag in alphanumeric format.               |
+| Field | Description                                                         |
+|-------|---------------------------------------------------------------------|
+| count | The count field represents the number of NFTs with the current tag. |
+| tag   | This field represents the tag in a alphanumeric format.             |
```
Comment (mariusmihaic) [inline docs/sdk-and-tools/indices/tags.md:16]:
```suggestion
| tag   | This field represents the tag in an alphanumeric format.             |
```

File: docs/sdk-and-tools/indices/delegators.md (hunk 16 above)
Comment (miiu96) [reply docs/sdk-and-tools/indices/delegators.md:18]: changed

File: docs/sdk-and-tools/indices/tags.md (hunk 17 above)
Comment (miiu96) [reply docs/sdk-and-tools/indices/tags.md:16]: done

**Review by mariusmihaic — APPROVED** [review]

**Pull request merged**